app.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "MJML API",
+  "description": "Responsive email as a service. This project exposes mjml as an API service. It's meant* to be compatible with MJML API .",
+  "repository": "https://github.com/vergissberlin/mjml-api",
```

```suggestion
  "repository": "https://github.com/baethon/mjml-api",
```

I don't want to sound like ass, but I think we should point to the origin repo :)